.bin/create-rust-webpack.js

```diff
@@ -1,16 +1,26 @@
 #!/usr/bin/env node
 
 const { spawn } = require("child_process");
+const fs   = require("fs");
 
-const clone = spawn("git", ["clone", "https://github.com/rustwasm/rust-webpack-template.git", "."]);
+let folderName = '.';
+
+if(process.argv.length >= 3)  {
```

Comment:
nitpick: add a space after "if" before "(" and remove the double space after ")" and before "{" so it is just a single space.